curve_apps/trend_lines/params.py

```diff
@@ -23,7 +23,7 @@
 from curve_apps import assets_path
 
 
-class SourceParameters(BaseModel):
+class TrendLineSourceParameters(BaseModel):
```

Comment:
Same here, could make the class name shorter and avoid repeating the module name.